management-cluster-access/README.md

```diff
@@ -0,0 +1,95 @@
+# Ensure no single point of failure in management cluster access
+
+## Access to MCs for giant swarm staff in vintage vs the CAPI product
+
+Our default access method to management clusters is now single-sign-on.
+For Giant Swarm staff this is done via `dex` using github as the _only_ identity provider. 
+
+Our vintage product also offers the possibility to create a client certificate to access the management cluster.
+This means that in the event of github or `dex` being unavailable, we do not get locked out of the management cluster.
+
+For pure CAPI installations on the other hand, no other authentication method is available at this point.
```

Comment:
There's a CAPi kubeconfig that gets created on MC creation and that we store in Lastpass AFAIK